routes/users.js

```diff
@@ -34,30 +34,38 @@ router.post('/register', function(req, res){
 	var errors = req.validationErrors();
```

Comment:
It is used on Line 32, it essentially works as the 'confirm' password, as seen on other sites, to ensure the user has typed it in correctly the first time